config/routes.rb

```diff
@@ -10,11 +10,10 @@
   # Defines the root path route ("/")
   # root "posts#index"
   root "standings#index"
+
+  resources :simulations, only: %i[index create new show]
+
   get "/matches", to: "matches#index"
```

Comment:
Similarly, you can (and, conventionally, should) change this to:

```rb
resources :matches, only: %i[index]
```

It simplifies the routes file and keeps things consistent. The direct mapping is more appropriate if you're defining custom routes or breaking conventions - but, IMO, that is also a sign that maybe you're writing your controllers out of the conventional style, too.

Reply:
Ack. Done in #17